pkgs/tools/security/posteid-seed-extractor/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication {
+  pname = "posteid-seed-extractor";
+  version = "unstable-2022-03-22";
```

Comment:
The commit date of 667e299 is 2022-02-23, I think you flipped the digits

Reply:
Ooops, fixed :)